src/utils/pdf-export.ts

```diff
@@ -83,6 +86,11 @@ export class PDFExport {
 			}
 		}
 
+		const domains = features.find(f => f.type == FeatureType.Domain)?.data?.selected as Domain[];
```

Comment:
Does this really solve the issue? You're looking for a `FeatureType.Package`

Reply:
It does solve it, the "Domains" seem to be stored in a "Domain Feature Choice" feature.

Reply:
Correction, a "Domain" feature of type "Domain"